treeherder/model/models.py

```diff
@@ -188,12 +189,17 @@ class Commit(models.Model):
 
     class Meta:
         db_table = "commit"
-        unique_together = ("push", "revision")
+        unique_together = ("push", "revision"),
```

Comment:
I think you don't need this comma at the end (this is python, not javascript :-) )

Reply:
haha very true